src/Components/ModuleForm/ModuleForm.js

```diff
@@ -0,0 +1,310 @@
+import React, { Component } from "react";
+import "./ModuleForm.css";
+import { Button, ListGroup, Card } from "react-bootstrap";
+
+class ModuleForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      enabled: "1",
+      userAction: "Add",
+      moduleName: "",
+      moduleList: JSON.parse(localStorage.getItem("moduleList")) || [],
+      grade: "A",
+      error: "",
+    };
+  }
+
+  handleSubmit = (event) => {
+    const { userAction, moduleList, moduleName, grade } = this.state;
+
+    console.log("handleSubmit");
```

Comment:
never keep console logs in your pushed code. Especially this one serves no purpose